app/code/AKisilenko/ModuleLesson6/Api/AskQuestionRepositoryInterface.php

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace AKisilenko\ModuleLesson6\Api;
+
+use AKisilenko\ModuleLesson6\Api\Data\AskQuestionInterface;
+use Magento\Framework\Api\SearchCriteriaInterface;
+
+/**
+ * Interface AskQuestionRepositoryInterface
+ * @package AKisilenko\ModuleLesson6\Api
+ */
+interface AskQuestionRepositoryInterface
+{
+    /**
+     * @param Data\AskQuestionInterface $askQuestion
+     * @return mixed
+     */
+    public function save(AskQuestionInterface $askQuestion);
+
+    /**
+     * @param $askQuestionId
+     * @return mixed
+     */
+    public function getById($askQuestionId);
+
+    /**
+     * @param SearchCriteriaInterface $searchCriteria
+     * @return mixed
```

Comment:
return type of getList method should be not mixed but:
```
* @return \Magento\Framework\Api\SearchResults
```
Type hinting also need to be checked, but API used annotations as information about what data is returned